Resources/Prototypes/InventoryTemplates/smile_inventory_template.yml

```diff
@@ -0,0 +1,9 @@
+- type: inventoryTemplate
```

Comment:
Check another inventory templates
100% you can find that one, looking exactly same (with only head slot)

Reply:
It does, forgot about it when removing the comments